modules/nw-operator-cr.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-network-operator.adoc
```

Comment:
- In configuring-network-operator.adoc, the annotations make it sound like you can still modify some of the values when you can't.

Reply:
Copied to Google Doc to visualize in context.